components/gantry/oneaxis/oneaxis.go

```diff
@@ -101,8 +85,8 @@ type oneAxis struct {
 
 	limitSwitchPins []string
 	limitHigh       bool
-	limitType       limitType
 	positionLimits  []float64
+	posRange        float64
```

Comment:
I don't like mixing variables that contain the abbreviated and non-abbreviated form of the same word. How about `positionRange`, to go with `positionLimits`? It makes the code style more consistent.